README.md

```diff
@@ -77,6 +77,24 @@ this driver supports the following:
 > **Note**
 > `msedgedriver` support is limited. `appium:autodownloadEnabled` does not work for the driver, thus `appium:executable` is necessary to automate MSEdge browser properly.
 
+## Scripts
+
+### install-chromedriver
+
+This script is used to install the given or latest stable version of chromedriver (113+) from
```

Comment:
Probably we should also link a tutorial here on how to properly select chromedriver for the given chrome browser version

Reply:
What about right now?